CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased](https://github.com/FakerPHP/Faker/compare/v1.20.0...main)
 
+## [2022-12-13, v1.21.0](https://github.com/FakerPHP/Faker/compare/v1.20.0..v1.21.0)
```

Comment:
Not sure, is there anything else we need to do here - given that we want to tag a new release today?

Reply:
I don't think so, this is the only user-land changes we made.